### ISSUE TITLE
Upgrading to acquia_cms v1.5.2 downgrades all dependent modules.

### ISSUE THREAD

**Issue by thegbomb** [acquia/acquia_cms#1480]
**Describe the bug**
When following the instructions to upgrade Acquia CMS at:
https://docs.acquia.com/acquia-cms/upgrading/
the installation cannot find an installable set of packages and in many cases it downgrades the versions of modules installed.

**To Reproduce**
1. Run: composer update acquia/acquia_cms:^1.5 -W --dry-run
2. View the output:
` - Downgrading drupal/acquia_cms_article (1.4.0 => 1.3.5)
  - Downgrading drupal/acquia_cms_audio (1.4.0 => 1.3.2)
  - Downgrading drupal/acquia_cms_common (1.5.0 => 1.3.11)
  - Downgrading drupal/acquia_cms_component (1.3.3 => 1.3.2)
  - Downgrading drupal/acquia_cms_document (1.4.0 => 1.3.3)
  - Downgrading drupal/acquia_cms_event (1.4.0 => 1.3.5)
  - Downgrading drupal/acquia_cms_image (1.4.0 => 1.3.6)
  - Downgrading drupal/acquia_cms_page (1.4.0 => 1.3.6)
  - Downgrading drupal/acquia_cms_person (1.4.1 => 1.3.5)
  - Downgrading drupal/acquia_cms_place (1.4.0 => 1.3.6)
  - Downgrading drupal/acquia_cms_search (1.4.2 => 1.3.8)
  - Downgrading drupal/acquia_cms_site_studio (1.4.1 => 1.3.8)
  - Downgrading drupal/acquia_cms_starter (1.3.3 => 1.3.2)
  - Downgrading drupal/acquia_cms_toolbar (1.4.0 => 1.3.3)
  - Downgrading drupal/acquia_cms_tour (1.3.3 => 1.3.1)
  - Downgrading drupal/acquia_cms_video (1.4.0 => 1.3.3)
`

**Expected behavior**
I expect these modules to be upgraded to the latest versions available
2. View the output:
`  - Upgrading drupal/acquia_cms_article (1.4.0 => 1.5.4)
  - Upgrading drupal/acquia_cms_common (1.5.0 => 2.0.3)
  - Upgrading drupal/acquia_cms_document (1.4.0 => 1.4.3)
  - Upgrading drupal/acquia_cms_event (1.4.0 => 1.5.4)
  - Upgrading drupal/acquia_cms_image (1.4.0 => 1.5.6)
  - Upgrading drupal/acquia_cms_page (1.4.0 => 1.5.4)
  - Upgrading drupal/acquia_cms_person (1.4.1 => 1.5.4)
  - Upgrading drupal/acquia_cms_place (1.4.0 => 1.5.4)
  - Upgrading drupal/acquia_cms_search (1.4.2 => 1.4.9)
  - Upgrading drupal/acquia_cms_site_studio (1.4.1 => 1.5.6)
  - Upgrading drupal/acquia_cms_toolbar (1.4.0 => 1.4.3)
  - Upgrading drupal/acquia_cms_tour (1.3.3 => 2.1.5)
  - Upgrading drupal/acquia_cms_video (1.4.0 => 1.5.4)
`

**Comment by rajeshreeputra** [comment]
Could you please check with `composer why-not drupal/acquia_cms_common ^2.0` and fix the root composer.json(if theres any indirect dependencies added in root composer.json file which causing problem to upgrade acquia_cms to latest)?

**Comment by thegbomb** [comment]
Do we not need to go to 1.5 first to be able to run `drush acms:switch` ?

**Comment by vishalkhode1** [comment]
@thegbomb Yes. you need to go to latest release of 1.5.x to run drush acms:switch

> Do we not need to go to 1.5 first to be able to run `drush acms:switch` ?
